docs/README.md

```diff
@@ -29,17 +28,19 @@ To begin, get familiar with the kit provided. It includes : <!--StartFragment-->
 | 2﻿     | M﻿otor                     |
 | 2﻿     | W﻿heels                    |
 | 1﻿     | C﻿aster Wheel              |
-| 1﻿     | C﻿hassis Kit               |
+| 1﻿     | C﻿hassis              |
 | 2﻿     | S﻿ervo Motor               |
 | 1﻿     | S﻿tart Button              |
 | 1﻿     | P﻿ower Switch              |
 
-:﻿:: caution- Handling batteries incorrectly can be dangerous. Make sure you follow these [safety guidelines](/docs/charging.html) at all times. :::
+:::warning 
+ Handling batteries incorrectly can be dangerous. Make sure you follow these [safety guidelines](/docs/charging.html) at all times.
+:::
 
 :::tip
 You can also access these docs from your robot. Go to <http://robot.go/docs> when you're connected to your robot.
 :::
 
-It might also be a good idea to sign up for the [forum](/forum/). You can do this using the registration code provided in your kit. This is a great place to ask questions on anything related to the competition, the kit, or robot programming.
+It might also be a good idea to sign up for the [forum](/forum/).  This is a great place to ask questions on anything related to the competition, the kit, or robot programming.
```

Comment:
is the forum still up & can you sign up? Did we provide codes in their kit? If so, we should say that. If not, we should remove this entirely